sdk/python/setup.py

```diff
@@ -66,6 +66,9 @@
 ]
 
 GCP_REQUIRED = [
+    # proto-plus>=1.19.7 requires protobuf>=3.19.0
+    # which conflicts with the version required by google-api-core[grpc]
```

Comment:
While those comments are really useful in the PR, it should be removed in the code

Reply:
Deleted by 42d6f6f.